src/static/css/picker.css

```diff
@@ -172,10 +172,14 @@
 	left: 1em;
 	background: white;
 	border: 1px solid var(--main-border);
+	border-bottom: 2px solid var(--main-border);
```

Comment:
Hmm do you have any idea to make it feel like a button and look nice with the tab group container?